app/index.html

```diff
@@ -7,11 +7,11 @@
     <link rel="SHORTCUT ICON" href="./favicon.svg" type="image/x-icon">
     
     <!-- msal.min.js can be used in the place of msal.js; included msal.js to make debug easy -->
-    <script src="https://alcdn.msauth.net/browser/2.3.0/js/msal-browser.js" integrity="sha384-ILJg8BOvXQwFGYEbkLVLYTYoNpTT7tP905UubLu2AqwksVdddAu5z9k3e6gMhqc5" crossorigin="anonymous"></script>
+    <script src="https://alcdn.msauth.net/browser/2.6.1/js/msal-browser.js" integrity="sha384-kHVR+hnKKUXpL5UEI3dgmdIKZgopBagC1RdQytFqglEGROvOSAGJRkaFWfu8VsSx" crossorigin="anonymous"></script>
```

Comment:
This is up to 2.7.0 now, our readmes have not been updated yet but you can find the SRI hashes in this PR https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/2613

Reply:
ah, didn't realize -done!